docling/models/base_ocr_model.py

```diff
@@ -31,6 +33,8 @@ def __init__(
         options: OcrOptions,
         accelerator_options: AcceleratorOptions,
     ):
+        from scipy.ndimage import binary_dilation, find_objects, label
```

Comment:
Judging from the test failures, this import line should be moved into `def get_ocr_rects` below.